crates/aptos/src/move_tool/mod.rs

```diff
@@ -293,12 +312,33 @@ impl CliCommand<()> for InitPackage {
             .map(|(key, value)| (key, value.account_address.into()))
             .collect();
 
+        // Initialize move directory
+        let package_dir_path = package_dir.as_path();
         self.framework_package_args.init_move_dir(
-            package_dir.as_path(),
+            package_dir_path,
             &self.name,
             addresses,
             self.prompt_options,
-        )
+        )?;
+
+        // Add in any template associated
+        match self.template {
+            Some(Template::HelloBlockchain) | None => {
+                // Setup the Hello blockchain template
+                write_to_file(
+                    package_dir_path.join("Move.toml").as_path(),
+                    "Move.toml",
+                    HELLO_BLOCKCHAIN_MOVE_TOML.as_bytes(),
+                )?;
+                write_to_file(
+                    package_dir_path
+                        .join("sources/hello_blockchain.move")
```

Comment:
So sources already exists at this point yeah?

Reply:
init_move_dir creates the directory.